src/domains/access.ml

```diff
@@ -49,7 +49,25 @@ struct
   let pretty () x = pretty_f short () x
 end
 module LSSet = SetDomain.Make (LabeledString)
-module LSSSet = SetDomain.Make (LSSet)
+module LSSSet =
+struct
+  include SetDomain.Make (LSSet)
+  (* TODO: is this actually some partition domain? *)
+  let join po pd =
+    let mult_po s = union (map (LSSet.union s) po) in
+    fold mult_po pd (empty ())
+  let bot () = singleton (LSSet.empty ())
+  let is_bot x = cardinal x = 1 && LSSet.is_empty (choose x)
+  (* top & is_top come from SetDomain.Make *)
+
+  let leq _ _ = raise (Lattice.Unsupported "LSSSet.leq")
+  let meet _ _ = raise (Lattice.Unsupported "LSSSet.meet")
```

Comment:
Can this not happen as a result of having two different paths when path sensitivity is enabled?
Then the query results are `joined` i.e. `meet` is called for a reversed lattice.
If this can not happen, we should have a comment here why it can't.

Reply:
`meet` (i.e. `join` of `PartAccessResult`) isn't really needed because these queries are only happening in `MCP2` and never go up to the path sensitivity functor level, i.e. `Queries.PartAccess` isn't ever made from outside. Just like `part_access` wasn't previously used or properly implemented for the `Spec` functors.

I also added that as a comment to the code.